contrib/babelfishpg_tsql/sql/information_schema_tsql.sql

```diff
@@ -234,13 +234,18 @@ $$SELECT
 
 
 /*
- * COLUMNS view
+ * COLUMNS view internal
  */
 
-CREATE OR REPLACE VIEW information_schema_tsql.columns AS
-	SELECT CAST(nc.dbname AS sys.nvarchar(128)) AS "TABLE_CATALOG",
+CREATE OR REPLACE VIEW information_schema_tsql.columns_internal AS
+	SELECT a.attrelid AS "TABLE_OID",
+			CAST(nc.dbname AS sys.nvarchar(128)) AS "TABLE_CATALOG",
 			CAST(ext.orig_name AS sys.nvarchar(128)) AS "TABLE_SCHEMA",
-			CAST(c.relname AS sys.nvarchar(128)) AS "TABLE_NAME",
+			CAST(
+				CASE WHEN c.reloptions[1] LIKE 'bbf_original_rel_name%' THEN substring(c.reloptions[1], 23)
+            	ELSE c.relname END
```

Comment:
Is this change really necessary? Because, pg can store the original object name if created using "", in that case the previous condition will give us correct results. 
Although, tsql returns original object name when queried whereas we are returning the downcase version of it. If we use this change in every such occurrence, we can return the original name of object from tds side. IMO, we should have a discussion regarding this.

Reply:
ISC.tables returns the original name of table, we should be returning the original name here also this will also fix issue occuring due to cross-dialect tables. Lets discuss this with Dipesh.

Reply:
Yes, just to need to decide if we want to fix this here, or take this separately.
There are some other columns which need this fix in ISC schema.